Java-Introduction.md

```diff
@@ -0,0 +1,125 @@
+# Introduction
+
+## What is Java?
+
+[Java](https://www.oracle.com/java/index.html) is a programming language developed by [Sun Microsystems](https://en.wikipedia.org/wiki/Sun_Microsystems) in 1995, which got later acquired by [Oracle](http://www.oracle.com/index.html). It's now a full platform with lots of standard APIs, open source APIs, tools, huge developer community and is used to build the most trusted enterprise solutions by big and small companies alike. [Android](https://www.android.com/) operating system is built fully with Java and its ecosystem. To know more about Java, read [this](https://java.com/en/download/faq/whatis_java.xml) and [this](http://tutorials.jenkov.com/java/what-is-java.html).
```

Comment:
> [Android](https://www.android.com/) operating system is built fully with Java and its ecosystem.

Android at its kernel level is a fork of Linux. It's built with Linux kernel, some C/C++ wrappers on top of it  and the ART. Android app development is done in Java; but Android itself is not written in Java.